src/JitInterface/src/CorInfoBase.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Internal.JitInterface
 {
-    public unsafe partial class CorInfoImpl
+    unsafe partial class CorInfoImpl
```

Comment:
Nit - could we be consistent and add the internal visibility modifier here too?

Reply:
This file is auto-generated so it would get eaten next time we take a new Jit. The place to fix this is where we generate this file.

Reply:
Ah, I see, my bad, I'm still learning :-).